js-webapp/README.md

```diff
@@ -164,7 +168,8 @@ Now we're ready to move on to the WebAssembly part.
 
 ### Binding
 
-The binding of the C++ code:
+To use the Newton-Raphson code from JavaScript, we'll need to define the _bindings_ file. The binding provides entry
+points to C++ methods and classes, which can then be used by JavaScript (after compilation). For our Newton-Raphson code, the binding file looks like this:
```

Comment:
It is hard to describe what binding does. Entry points are a bit vague to me.

Some alternatives
* The binding allows to map C++ constructs to their JavaScript equivalent
* The binding allows compiled code to be used in a natural way in JavaScript
* The binding allows compiled code to be called from JavaScript
* The binding provides method to connect and interact between JavaScript and compile C++